pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -12,27 +12,99 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import os
+from pprint import pprint
+from typing import Iterable, Union, cast
+from copy import deepcopy
+from collections import ChainMap
 import torch
 from pytorch_lightning.core import memory
 from pytorch_lightning.loggers import TensorBoardLogger, LoggerCollection
 from pytorch_lightning.utilities import flatten_dict
 from pytorch_lightning.utilities.model_utils import is_overridden
 from pytorch_lightning.core.step_result import EvalResult, Result
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pprint import pprint
-from typing import Iterable
-from copy import deepcopy
-from collections import ChainMap
+from pytorch_lightning.trainer.connectors.logger_connector.callback_hook_validator import CallbackHookNameValidator
+from pytorch_lightning.trainer.connectors.logger_connector.epoch_result_store import (
+    EpochResultStore,
+    LoggerStages,
+    LOOKUP_TABLE
+)
 
 
 class LoggerConnector:
 
+    __lookup_stages = LOOKUP_TABLE
```

Comment:
why do we need them as static attribute? can't we just use the imported global variable?